lexos/templates/scrub.html

```diff
@@ -34,6 +34,7 @@
                 }
                                 });
            tagDict=JSON.stringify(tagDict);
+           console.log(tagDict);
```

Comment:
I think vscode has chrome debuggers, I am not sure it is great. I am pretty positive sublime text will also have chorme debugger and firefox debugger.